### ISSUE TITLE
create a cache of document titles 

### ISSUE THREAD

**Issue by sknorr** [openSUSE/docserv#18]
to generate an overview page... we need one of the following xml files per docset. generate these files into a directory that is semi-temporary (i.e. not deleted after every build, possibly location configurable via the ini file).

:
```
<docset productid="sled" setid="15ga" lang="en-us">

  <document dc="DC-SLES-all" rootid="book.security" format="pdf" hash="MD5*">
    <commit>faabbaaf</commit>
    <title>Security Guide</title>
    <path>en-us/sles/15ga/book.security/book.security_color_en.pdf</path>
  </document>

  <document ...>
   ...
  </document>
</docset>
```

\* MD5 of the following string:

* MAIN file +
* root ID +
* profiling attributes from DC file, concatenated, spaces removed from values (e.g. `PROFARCH=x86_64;power PROFCONDITION=beta`)

**Comment by svenseeberg** [comment]
Required:
* d2d_runner must be able to build the big file along with other targets, for example with --bigfile=1
* use regex to extract root ID from DC file
* use root ID to get text from title tag out of big file

**Comment by sknorr** [comment]
As discussed, the path for these files should be:

```
/var/cache/docserv/[CONFIG-INI_FILE_NAME]/[la-ng]/[PRODUCT_ID]/[SET_ID].xml
```

**Comment by svenseeberg** [comment]
@sknorr what is the target of the overview page? the root of each language? like in /en-us/overview.html and then the docs are in /en-us/product/docset/stuff ?

**Comment by svenseeberg** [comment]
Fixed in #23
